app/components/modal-error/error.modal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import { Modal, View, StyleSheet, Dimensions, TouchableOpacity} from 'react-native';
+import VectorIcon from 'react-native-vector-icons/FontAwesome';
+import { color } from '../../theme';
+import { Text } from '../text/text';
+const { width: SCREEN_WIDTH } = Dimensions.get('window');
+const ALERT_WIDTH = SCREEN_WIDTH - (SCREEN_WIDTH / 4);
+
+interface IErrorModalProps {
+	visible: boolean,
+	message: string,
+	handelClose: Function
+}
+
+
+class ErrorModal extends Component<IErrorModalProps> {
+	constructor (props: IErrorModalProps) {
+		super(props);
+	}
+
+	render() {
+		return (
+			<Modal visible={this.props.visible} animationType="fade" transparent onRequestClose={() => null }>
+				<View style={styles.container}>
+					<View style={[styles.iconCircle, { top: 32, borderColor: color.palette.white }]}>
+						<View style={styles.iconCircleTop}>
+							<VectorIcon name="close" size={20} color='white' />
+						</View>
```

Comment:
This view component repeats itself (e.g line 33). so basically we would like to export it to its own component and reuse it.

Reply:
really? you would do a dedicated component for a circle? 
if you tell me a trust you, but specifically i got a new UX/UI from Doron so i will need to change the design anyway, and most of this UI won't be relevant